packages/extension-ui/src/components/PasswordField/getFeedback.ts

```diff
@@ -16,23 +16,39 @@ const APP_RELATED_INPUTS = ['aleph', 'zero', 'aleph zero', 'signer'];
 export const getUserInputs = (name?: string | null) => {
   const nameRelatedInputs = name ? [name, ...name.split(/[^a-zA-Z]+/)] : [];
 
-  return [
-    ...APP_RELATED_INPUTS,
-    ...nameRelatedInputs,
-  ];
+  return [...APP_RELATED_INPUTS, ...nameRelatedInputs];
 };
 
-export const isPasswordTooWeak = (feedback: ValidationResult) =>  {
+export const isPasswordTooWeak = (feedback: ValidationResult) => {
   return feedback.score < LOWEST_ACCEPTED_PASSWORD_SCORE;
 };
 
+const nonLatinAlphanumeric = /\W/;
+
+const addPunctuationIfMissing = (sentence: string) => {
+  const lastChar = sentence.at(-1);
+
+  if (!lastChar) {
```

Comment:
When would the last char be falsy? Does that basically mean the `sentence` is an empty string?

Reply:
yes, it's mostly "typescript not complaining" thing